.drone.yml

```diff
@@ -32,6 +37,48 @@ steps:
     commands:
       - ./gradlew --no-daemon dependencyUpdates
 
+  - name: dev - assume role
+    image: amazon/aws-cli
+    environment:
+      ROLE_ARN:
+        from_secret: dev_role_arn
+    commands:
+      - mkdir -p .aws
+      - |-
+        cat << EOF > .aws/credentials
+        [default]
+        role_arn = $${ROLE_ARN}
+        role_session_name = ${DRONE_COMMIT_SHA:0:8}_${DRONE_BUILD_NUMBER}
+        credential_source = Ec2InstanceMetadata
+        EOF
+
+  - name: dev - release
+    image: amazon/aws-cli
+    environment:
+      BUCKET_NAME:
+        from_secret: dev_bucket
+    commands:
+      - cp -r .aws $HOME/.aws
+      - echo "$${BUCKET_NAME} a${BUCKET_NAME} $BUCKET_NAME"
```

Comment:
what's this echo doing? also there is a random 'a' in there

Reply:
It was for testing. Tidied it up.